src/Conticket/Model/Aggregates/Event/Event.php

```diff
@@ -53,4 +60,23 @@ public function whenEventWasCreated(EventWasCreated $eventWasCreated)
         $this->name = $eventWasCreated->name();
         $this->description = $eventWasCreated->description();
     }
+
+    public function addTicket($name, $description)
```

Comment:
Let's suppose that I receive a Ticket as parameter instead of name and description, is it ok that the Ticket aggregate has property accessors as Ticket::name() and Ticket::description()? I thought Aggregates, and I started modeling Ticket as aggregate, shouldn't have property accessor besides its aggregateId()